apps/api/src/app/endpoints/watchlist/watchlist.service.ts

```diff
@@ -97,6 +102,35 @@ export class WatchlistService {
       where: { id: userId }
     });
 
-    return user.watchlist ?? [];
+    const quotes = await this.dataProviderService.getQuotes({
+      items: user.watchlist.map(({ dataSource, symbol }) => {
+        return { dataSource, symbol };
+      }),
+      requestTimeout: ms('30 seconds'),
+      useCache: false
```

Comment:
Remove `requestTimeout` and `useCache` (benchmarks is different because its full view is cached)